includes/admin-functions.php

```diff
@@ -73,7 +73,7 @@ function dokan_admin_shop_order_edit_columns( $existing_columns ) {
 function dokan_shop_order_custom_columns( $col ) {
     global $post, $the_order;
 
-    if ( empty( $the_order ) || dokan_get_prop( $the_order, 'id') != $post->ID ) {
```

Comment:
Remove `dokan_get_props` since we don't provide support below WC 3.0 version. Just call directly `$the_order->get_id()`